seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -691,7 +691,9 @@
       ],
       "tests": [
         "assert(quotient === 2.2, 'message: The variable <code>quotient</code> should equal <code>2.2</code>.');",
-        "assert(/\\//.test(code), 'message: You should use the <code>/</code> operator');"
+        "assert(/4\\.40*\\s*\\/\\s*2\\.*0*/.test(code), 'message: You should use the <code>/</code> operator.');",
+        "assert(code.match(/quotient/g).length === 3, 'message: The quotient variable should only be assigned once.');"
```

Comment:
TravisCI might be angry at this line. This is checking for 3 instances of `quotient`, but the solution should only have 1 instance of `quotient`.

![image](https://cloud.githubusercontent.com/assets/2754821/21296537/1e9799a8-c523-11e6-9a40-b3c4adda7e36.png)

So the test should be `code.match(/quotient/g).length === 1`, unless I'm missing something.